### PR TITLE
Dev 56

### PR DESCRIPTION
**Pull request by mharoonm**
Some UI fixes and Large Folder Name Truncation, Starred function in Mail Detail View

### CODE REVIEW

File: src/app/mail/mail-detail/mail-detail.component.html [hunk 1]
```diff
@@ -248,21 +248,24 @@ <h5 class="ui-header-subtitle text-dark mb-0">
...
   <!--Reply to mail.-->
-  <div class="ui-body" #replyActions>
+  <div class="ui-body ui-footer-sec" #replyActions>
 
     <div class="row">
       <div class="col-12">
 
         <button class="btn btn-gmail" (click)="onReply(mail)">
           <i class="icon icon-reply position-left"></i> Reply
+          <img src="assets/images/group-5.svg" class="hidden-sm-up">
```
[the remark this answers is not in the document]
Comment (mharoonm) [reply src/app/mail/mail-detail/mail-detail.component.html]: Fixed.

File: src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.html [hunk 2]
```diff
@@ -252,6 +252,71 @@
         </td>
 
       </tr>
+      <tr class="hidden-sm-up mail-list-row" *ngFor="let mail of mails; let i=index"
```
[the remark this answers is not in the document]
Comment (mharoonm) [reply src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.html]: To match mobile design

File: src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.html [hunk 3]
```diff
@@ -166,7 +166,7 @@
         <p class="py-2 mb-0">There are no conversations with this label.</p>
         <hr class="mt-0">
       </tr>
-      <tr class="mail-list-row" *ngFor="let mail of mails; let i=index"
+      <tr class="mail-list-row HideOnMobile" *ngFor="let mail of mails; let i=index"
```
[the remark this answers is not in the document]
Comment (mharoonm) [reply src/app/mail/mail-list/mail-folder/generic-folder/generic-folder.component.html]: Fixed.

File: src/app/mail/mail.component.scss [hunk 4]
```diff
@@ -1206,6 +1210,50 @@
       white-space: nowrap;
     }
   }
+  @include media("<sm") {
+    .fancy-field-group{
+      position: absolute;
```
[the remark this answers is not in the document]
Comment (mharoonm) [reply src/app/mail/mail.component.scss]: Fixed.
No need to remove this line.

File: src/styles/modules/mail-composer/_mail-composer.scss [hunk 5]
```diff
@@ -368,6 +381,11 @@ a {
 		font-size: 0.75rem;
 		min-width: 85px;
 	}
+    .dropdown-menu{
+        top: 43px !important;
```
[the remark this answers is not in the document]
Comment (mharoonm) [reply src/styles/modules/mail-composer/_mail-composer.scss]: There is some inline styles, To overwrite we have to use "!mportant"
screenshot

[the remark this answers is not in the document]
Comment (atifsaddique211f) [reply src/styles/modules/mail-composer/_mail-composer.scss]: ok

[the remark this answers is not in the document]
Comment (Waseem826) [reply src/styles/modules/mail-composer/_mail-composer.scss]: @mharoonm these inline styles are calculated and added by bootstrap. After your change the dropdown is not showing anymore

[the remark this answers is not in the document]
Comment (mharoonm) [reply src/styles/modules/mail-composer/_mail-composer.scss]: @Waseem826 Thanks for pointing out this. I fixed the Background issue in fix/Dev-347 branch.

**Pull request merged**